cmd/cosign/cli/options/certificate.go

```diff
@@ -75,12 +76,18 @@ func (o *CertVerifyOptions) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&o.CertGithubWorkflowRef, "certificate-github-workflow-ref", "",
 		"contains the ref claim from the GitHub OIDC Identity token that contains the git ref that the workflow run was based upon.")
 	// -- Cert extensions end --
+	cmd.Flags().StringVar(&o.CARoots, "ca-roots", "",
```

Comment:
Do we also want an ca-intermediates flag?

I think we should require either (--ca-roots + optionally --ca-intermediates) or --certificate-chain.

Reply:
I wasn't sure if we want both flags right away or to start with `--ca-roots` and follow up with `--ca-intermediates` if/when someone needs such combination.  I can add `--ca-intermediates` flag - should I do it in this PR or in a follow-up?

Reply:
With this implementation with only ca-roots, how can you provide chain building CA certificates? I would prefer to add both, since I think it makes documentation clearer - Either use the deprecated --certificate-chain, or use --ca-roots/intermediates

Reply:
ok, will add `--ca-intermediates` shortly.   Working to expand the functional tests for the intermediate case.
(With `--ca-roots` only, you can provide multiple CA roots certificates in a bundle file, but only "one level" roots, cannot have an intermediate certificate (with `CA: true` in the template) signed by the root certificate.)

Reply:
added `--ca-intermediates` and adjusted the functional test as well as the docs PR https://github.com/sigstore/docs/pull/291 - please take a look @haydentherapper